db/database.go

```diff
@@ -438,67 +436,62 @@ func NewDatabaseContext(ctx context.Context, dbName string, bucket base.Bucket,
 	// Initialize the tap Listener for notify handling
 	dbContext.mutationListener.Init(bucket.GetName(), options.GroupID)
 
-	if len(options.Scopes) > 0 {
-		dbContext.Scopes = make(map[string]Scope, len(options.Scopes))
-		dbContext.CollectionNames = make(map[string]map[string]struct{}, len(options.Scopes))
-		for scopeName, scope := range options.Scopes {
-			dbContext.Scopes[scopeName] = Scope{
-				Collections: make(map[string]*DatabaseCollection, len(scope.Collections)),
+	if len(options.Scopes) == 0 {
```

Comment:
I don't want to do this, because I don't think that we often want to test with the default collection explicitly. There are a few circumstances (testing legacy attachment behavior, upgrade behavior) but mostly we want to be able to have our databases backed by collections determined by the test harness.

This is a summary of what was discussed offline.